src/routes/(app)/+layout.svelte

```diff
@@ -188,7 +189,8 @@
   </div>
   <MainContentContainer>
     <DataEncoderSettings class="hidden md:flex" />
-    <TopNavigation {namespaceList}>
+    <TopNavigation>
+      <NamespacePicker {namespaceList} slot="left" />
```

Comment:
Will this actually update with a new namespaceList (say when new ones are created and we bust the cache after a minute)? Since it's in a slot I wonder...